backend/src/main/java/de/neuefische/teddyscoronadiaries/openweatherapi/model/WeatherContent.java

```diff
@@ -0,0 +1,18 @@
+package de.neuefische.teddyscoronadiaries.openweatherapi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class WeatherContent {
+    @JsonProperty("main")
+    private String weatherClass;
+    @JsonProperty("description")
+    private String weatherDescription;
+    @JsonProperty("icon")
+    private String iconId;
+}
```

Comment:
`Content` is way to generic to help me understand what this data represents 🤔 I would not prefix the members with `weather` - I see the need for `weatherClass` as `class` is a Java keyword, but now I wonder what the class of weather is?

Reply:
the best class of weather is of course `Sunshine ☀️`

Reply:
Okay 😆